samri/pipelines/utils.py

```diff
@@ -398,6 +425,16 @@ def ss_to_path(subject_session):
 	return "/".join([subject,session])
 
 def bids_dict_to_source(bids_dictionary, source_format):
+	"""Generate a string from the source format with inserted corresponding values from the given dictionary.
+
+	Parameters
+	----------
+
+	bids_dictionary : dict
+		A dictionary with keys that are BIDS identifiers.
+	source_format : str
+		A source string with placeholders.
```

Comment:
A Python format string (that's the name under which you can most easily find info via web search) with identifiers which are keys of the `bids_dictionary` parameter.